soh/soh/GameMenuBar.cpp

```diff
@@ -236,8 +334,6 @@ namespace GameMenuBar {
                     UIWidgets::Tooltip("Allows windows to be able to be dragged off of the main game window. Requires a reload to take effect.");
                 }
 
-                EXPERIMENTAL();
```

Comment:
Since we're not using this call anywhere anymore, you can also remove its definition.